Description: Upstream changes introduced in version 4.6.0-0ubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 unity (4.6.0-0ubuntu1) oneiric; urgency=low
 .
   * New upstream release.
     - compiz crashed with SIGSEGV in __strlen_sse2() (LP: #814619)
     - PlacesHomeView::PlacesHomeView leaks memory (LP: #818450)
     - PluginAdapter::MaximizeIfBigEnough leaks memory (LP: #818477)
     - Launcher - Make Launcher left of screen reveal more responsive and less
       prone to false positives (LP: #765819)
     - Launcher - clicking on a App launcher icon incorrectly un-minimizes
       windows (LP: #783434)
     - Unity doesn't get any mouse wheel scroll event in Indicators InputArea
       (LP: #814574)
     - Unity launcher gets cluttered when having multiple partitions and/or
       external volumes attached (LP: #713423)
     - Unity panel menus and indicators slow to respond. Too much lag.
       (LP: #742664)
     - In Unity the distinction between GVolume, GDrive and GMount is a bit
       confusing. (LP: #799890)
     - Launcher - When a item is deleted by dragging to Trash, the trash should
       pulse once before the Launcher disappears (LP: #750311)
     - ccsm needs an option to change launcher opacity (LP: #815032)
     - add a ccsm option to hide volumes in launcher (LP: #794707)
     - scale plugin doesn't work as desired when "Click Desktop To Show
       Desktop" is true (LP: #810315)
     - mute/unmute sound when user clicks on sound applet using scroll button
       or middle mouse button (LP: #609860)
     - Secondary activate (i.e. middle click) support for indicators advanced
       usage (LP: #812933)
   * debian/control:
     - dep on latest libunity-misc
     - dep on latest nux
     - add build-dep on libgnome-desktop-3-dev
   * debian/rules:
     - bump libunity-core-4.0-4 shlib for ABI break
     - don't ship unity dialogs right now. Not ready for alpha quality
   * distro-patch the grey to darker grey (until the blur is done in nux)
   * Switch to dpkg-source 3.0 (quilt) format
   * debian/patches/01_revert_removed_function_for_unity2d_to_build.patch:
     - revert a removed API making unity-2d not building
 .
 The person named in the Author field signed this changelog entry.
Author: Didier Roche <didrocks@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/609860
Bug-Ubuntu: https://bugs.launchpad.net/bugs/713423
Bug-Ubuntu: https://bugs.launchpad.net/bugs/742664
Bug-Ubuntu: https://bugs.launchpad.net/bugs/750311
Bug-Ubuntu: https://bugs.launchpad.net/bugs/765819
Bug-Ubuntu: https://bugs.launchpad.net/bugs/783434
Bug-Ubuntu: https://bugs.launchpad.net/bugs/794707
Bug-Ubuntu: https://bugs.launchpad.net/bugs/799890
Bug-Ubuntu: https://bugs.launchpad.net/bugs/810315
Bug-Ubuntu: https://bugs.launchpad.net/bugs/812933
Bug-Ubuntu: https://bugs.launchpad.net/bugs/814574
Bug-Ubuntu: https://bugs.launchpad.net/bugs/814619
Bug-Ubuntu: https://bugs.launchpad.net/bugs/815032
Bug-Ubuntu: https://bugs.launchpad.net/bugs/818450
Bug-Ubuntu: https://bugs.launchpad.net/bugs/818477

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- unity-4.6.0.orig/services/CMakeLists.txt
+++ unity-4.6.0/services/CMakeLists.txt
@@ -39,7 +39,7 @@ set(CFLAGS
     "-DGETTEXT_PACKAGE=\"unity\""
     "-DINDICATORDIR=\"${_indicatordir}\""
     "-DINDICATORICONDIR=\"${_iconsdir}\""
-    "-Werror -Wall"
+    "-Wall"
     )
 add_definitions(${CFLAGS})
 
--- unity-4.6.0.orig/plugins/unityshell/src/BGHash.cpp
+++ unity-4.6.0/plugins/unityshell/src/BGHash.cpp
@@ -297,10 +297,10 @@ namespace unity {
     ubus_server_send_message(ubus_server_get_default(),
                              UBUS_BACKGROUND_COLOR_CHANGED,
                              g_variant_new ("(dddd)",
-                                            0.3,
-                                            0.3,
-                                            0.3,
-                                            0.9)
+                                            0.0,
+                                            0.0,
+                                            0.0,
+                                            0.95)
                                             //FIXME - using default colours for now
                                             //_current_color.red,
                                             //_current_color.green,
